### ISSUE TITLE
Unit Test Fails on Complex Equals/Select/Split Condition

### ISSUE THREAD

**Issue by aharshbarger** [DontShaveTheYak/cloud-radar#179]
A unit test fails to load a CFN template that utilizes a complex CFN condition in the `Conditions:` element that determines if a string value starts with a specific name.  

The CFN template functions as expected when deployed to AWS with the same complex condition.

```
AWSTemplateFormatVersion: 2010-09-09
Description: Sample of Complex Condition Using Select

Parameters:
  DBEngineVersion:
    Default: 5.7.mysql_aurora.2.10.2
    Type: String

Conditions:
  IsMySQL8:
    !Equals [ !Select [ 0, !Split [ "8.0.mysql_aurora", !Ref DBEngineVersion ] ], "" ]

  IsNotMySQL8:
    !Not [ Condition:  IsMySQL8 ]

Resources:
  SampleGroup57:
    Condition: IsNotMySQL8
    Type: AWS::RDS::DBClusterParameterGroup
    Properties:
      Description: "cluster group (mysql5.7)"
      Family: aurora-mysql5.7

  SampleGroup8:
    Condition: IsMySQL8
    Type: AWS::RDS::DBClusterParameterGroup
    Properties:
      Description: "cluster group (mysql8.0)"
      Family: aurora-mysql8.0
```
cloud-radar version: `0.6.3`
python version: `3.10.9`

Basic Unit test snippet:
```
    template: Template = request.getfixturevalue(templ_fix)
    stack = template.render(region = "us-west-2")

    assert "SampleGroup57" in stack["Resources"]
```
![image](https://user-images.githubusercontent.com/15339361/219394619-7265e5d5-1524-4e57-b6f8-655a1984ee24.png)

**Comment by shadycuz** [comment]
@aharshbarger 

According to the AWS documentation, you are not allowed to have a `!Select` function nested inside of a `!Equals` function. You can read about that [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-conditions.html#w4ab1c33c28c21c29). 

Unfortunately the AWS documentation is often wrong and there have been many occasions where I have had to expand on the allowed nested functions list inside my application with combinations that work but are undocumented. 

This is one such occasion and should be easy to fix.

**Comment by shadycuz** [comment]
Fixed in `0.6.4`
